src/api/app/views/webui/users/_breadcrumb_items.html.haml

```diff
@@ -10,7 +10,10 @@
       = @pagetitle.capitalize
   - when 'show'
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Profile of #{@displayed_user}
+      - if feature_enabled?('user_profile_redesign') && @is_displayed_user
```

Comment:
@saraycp IMO this doesn't need to be part of the beta program

Reply:
Well, maybe. Anyway, the `user_profile_redesign` is going to be rolled out in the upcoming days, I think it’s not worth changing it right now, in a few days will be out.

Reply:
Okay fine